### DISCUSSION TITLE
Github Actions UI highlights errors where it finds "[error]"

### DISCUSSION THREAD

**Discussion by mike-carey** [community/community#12623] [Actions]
When adding echo statements with "[error]" in them, the Github Actions UI marks the line as an error.  It marks both the source code, when you drop down the `run` section, and the output:

<img width="834" alt="Screen Shot 2022-03-09 at 6 37 01 PM" src="https://user-images.githubusercontent.com/32496966/157564108-d5cd50a2-deb0-485f-8e1b-d9b1a33e3150.png">

If too many of them are in one section, it really throws off the UI and it cannot even build the dropdown section:

<img width="725" alt="Screen Shot 2022-03-09 at 6 33 56 PM" src="https://user-images.githubusercontent.com/32496966/157564082-80307d64-2866-4873-8729-72345798cfc0.png">

This does not appear to be a Runner issue, I am guessing the frontend Javascript client is looking for `[error]` in the output.  I would consider this a bug, because I would expect the source code section (dropdown `run` section) not to be highlighted.

# References

* [Workflow](https://github.com/mike-carey/test/blob/main/.github/workflows/error-highlighting-issue.yml)
* [Completely broken UI run](https://github.com/mike-carey/test/actions/runs/1960457503)
* [Spaced out run](https://github.com/mike-carey/test/actions/runs/1960457503)

**Comment by ethomson** [comment]
Thanks for the feedback, @mike-carey - I'll pass your report along to the team.
